elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidatorTest.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.validator;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.exc.ValueInstantiationException;
+
+import org.apache.commons.cli.MissingArgumentException;
+import org.apache.commons.cli.MissingOptionException;
+import org.hjson.ParseException;
+import org.junit.jupiter.api.Test;
+
+public class DynamicConfigValidatorTest {
+
+    @Test
+    public void testNoArgumnents() {
+        Exception e = assertThrows(MissingOptionException.class, () -> DynamicConfigValidator.main(null));
+        assertTrue(e.getMessage().startsWith("Missing required option"));
+    }
+
+    @Test
+    public void testOneEmptyArgumnents() {
+        Exception e = assertThrows(MissingOptionException.class,
+                () -> DynamicConfigValidator.main(new String[] { "" }));
+        assertTrue(e.getMessage().startsWith("Missing required option"));
+    }
+
+    @Test
+    public void testMissingArgumnentValue() {
+        Exception e = assertThrows(MissingArgumentException.class,
+                () -> DynamicConfigValidator.main(new String[] { "--configDir" }));
+        assertTrue(e.getMessage().startsWith("Missing argument for option"));
+    }
+
+    @Test
+    public void testMissingConfigDir() {
+        Exception e = assertThrows(IllegalStateException.class, () -> DynamicConfigValidator
+                .main(new String[] { "--configDir", "src/test/resources/validator/missing" }));
+        assertEquals("Model Configs Directory doesn't exists", e.getMessage());
+    }
+
+    @Test
+    public void testValidConfigDir() {
+        assertDoesNotThrow(() -> DynamicConfigValidator
+                .main(new String[] { "--configDir", "src/test/resources/validator/valid" }));
+    }
+
+    @Test
+    public void testMissingVariableConfig() {
+        assertDoesNotThrow(() -> DynamicConfigValidator
+                .main(new String[] { "--configDir", "src/test/resources/validator/missing_variable" }));
+    }
+
+    @Test
+    public void testMissingSecurityConfig() {
+        assertDoesNotThrow(() -> DynamicConfigValidator
+                .main(new String[] { "--configDir", "src/test/resources/validator/missing_security" }));
+    }
+
+    @Test
+    public void testMissingTableDir() {
+        Exception e = assertThrows(IllegalStateException.class, () -> DynamicConfigValidator
+                .main(new String[] { "--configDir", "src/test/resources/validator/missing_table_dir" }));
+        assertTrue(e.getMessage().startsWith("Table Configs Directory doesn't exists at location"));
```

Comment:
The message is `Model Configs Directory doesn't exists`

Reply:
one empty director didnt get check in, checking it now to ensure it runs fine.